### PR TITLE
* Formatting 3d coordinates

### PR DESCRIPTION
**Pull request by fonsecareyna82**
* Formatting 3d coordinates in order to be used to compare them in different pseudosubtomograms

### CODE REVIEW

File: tomo/objects.py [hunk 1]
```diff
@@ -1056,7 +1056,7 @@ def copyInfo(self, other):
 
 class SetOfTomograms(data.SetOfVolumes):
     ITEM_TYPE = Tomogram
-    EXPOSE_ITEMS = True
+    EXPOSE_ITEMS = False
```
Comment (azazellochg) [inline tomo/objects.py:1059]: Why?

Comment (pconesa) [reply tomo/objects.py:1059]: I think is a question of having the ability to select a single tomogram in the set or not.

Do we already have this case?

Comment (pconesa) [reply tomo/objects.py:1059]: I think it is mainly to prevent this...
image

Comment (azazellochg) [reply tomo/objects.py:1059]: yes I'm using it to select single tomos in susantomo plugin

Comment (azazellochg) [reply tomo/objects.py:1059]: ah, no. nevermind, I'm using other objects.

**Comment by pconesa** [comment]
Ok, if we need it back we un do this.

**Review by pconesa — APPROVED** [review]
Thanks Yunior

**Pull request merged**